src/components/Form.js

```diff
@@ -593,6 +602,16 @@ class Form extends Component {
             Submit
           </Button>
         </ValidatorForm>
+        <Dialog
+          open={thanksOpen}
+          onClose={this.handleClose}
+        >
+          <DialogContent>
+            <DialogContentText>
+              Thank you for submitting! Your submission should appear on the map within a week.
+            </DialogContentText>
+          </DialogContent>
+        </Dialog>
```

Comment:
The dialogue box looks good to me. We could also send a screenshot of this to Holly to see if any more styling is needed.